php/templates/ingress.yaml

```diff
@@ -28,6 +28,9 @@ spec:
           {{- toYaml . | nindent 10 }}
           {{- end }}
           - path: {{ .path }}
+            {{- if semverCompare ">=1.19-0" $root.Capabilities.KubeVersion.GitVersion }}
+            pathType: ImplementationSpecific
```

Comment:
https://github.com/kubernetes/kubernetes/pull/89778/files#diff-d2017d343f877bd90044bde1cf563a4192386b918703b116d9135c1a312cc931R391
This type is only one of the `ImplementationSpecific`.